### PR TITLE
fix(cares,core,docs): fix Button style

### PR DESCRIPTION
**Pull request by eckdanny-osi**
and some other a11y nits. wip

📘 [`CL-189`](https://osi-cwds.atlassian.net/browse/CL-189)
🔗 [Preview Deploy](https://deploy-preview-546--cws-cares.netlify.com/components/button/demo)

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
I'm going to explode this into diff PR's... My machine had no network today so i just kept working on this branch. 😠

File: apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx]: lets start a file naming convention here...

```diff
- FooTearSheet.jsx
+ Foo.splat.jsx
```

File: apps/www/src/modules/components/routes.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import Badge from '!babel-loader!@mdx-js/loader!./articles/Badge/Badge.mdx'
 import Breadcrumbs from '!babel-loader!@mdx-js/loader!./articles/Breadcrumbs/Breadcrumbs.mdx'
 import BreadcrumbItem from '!babel-loader!@mdx-js/loader!./articles/BreadcrumbItem/BreadcrumbItem.mdx'
 import Button from '!babel-loader!@mdx-js/loader!./articles/Button/Button.mdx'
+import ButtonTearSheet from './articles/Button/ButtonTearSheet'
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/routes.js]: naming

File: apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Card,
+  CardTitle,
+  CardBody,
+  CardHeader,
+  Row,
+  Col,
+  Button,
+} from '@cwds/components'
+
+const PROP_COMBINATIONS = [
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx]: this is a nice format `combos: Array<[string, { [k:string]: any }]>`

File: apps/www/src/modules/components/routes.js [hunk 4]
```diff
@@ -59,6 +60,12 @@ const routes = {
       path: '/button',
       component: Button,
     },
+    {
+      path: '/button/demo',
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/routes.js]: pick a routing convention:

```diff
- path: '/foo/demo',
+ path: '/foo/splat'
```

File: libs/cares/src/Button/Button.jsx [hunk 5]
```diff
@@ -2,7 +2,8 @@ import React from 'react'
 import cn from 'classnames'
 import PropTypes from 'prop-types'
 import { Button as PrimitiveButton } from '@cwds/reactstrap'
-import Styles from './Button.module.scss'
+// import Styles from './Button.module.scss'
```
Comment (eckdanny-osi) [inline libs/cares/src/Button/Button.jsx]: clean up plz

File: libs/docs/src/Articles/ArticleSideNav.jsx [hunk 6]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
 import { ListGroup, ListGroupItem } from '@cwds/components'
+import Styles from './ArticleSideNav.module.scss'
```
Comment (eckdanny-osi) [inline libs/docs/src/Articles/ArticleSideNav.jsx]: this whole file can be removed from PR

File: libs/docs/src/Articles/ArticleSideNav.module.scss [hunk 7]
```diff
@@ -0,0 +1,10 @@
+@import "~@cwds/core/scss/base";
```
Comment (eckdanny-osi) [inline libs/docs/src/Articles/ArticleSideNav.module.scss]: this whole file can be removed from diff

File: apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+              <Row key={index} className="mb-2">
+                {/* <Col sm={2}>{name}</Col> */}
+                <Col>
+                  <Button {...props}>Button</Button>{' '}
+                  <Button {...props} active>
+                    Active
+                  </Button>{' '}
+                  <Button {...props} disabled>
+                    Disabled
+                  </Button>
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/articles/Button/ButtonTearSheet.jsx]: how about `focus` too?

File: libs/cares/src/Button/Button.module.scss [hunk 9]
```diff
@@ -4,22 +4,27 @@
   // Nothing Here
 }
 
-.Secondary {
-  @include button-variant($white, $primary);
-  color: $primary !important;
+// .Secondary {
```
Comment (eckdanny-osi) [inline libs/cares/src/Button/Button.module.scss]: 💩

**Comment by eckdanny-osi** [comment]
retest this please

File: apps/www/src/modules/components/articles/Button/Button.splat.jsx [hunk 10]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```
Comment (mlipardo) [inline apps/www/src/modules/components/articles/Button/Button.splat.jsx]: Tearsheets. 🎉

**Comment by eckdanny-osi** [comment]
added `release` label. Let's merge #550 and #551 1st

**Pull request merged**